07_iifo/index.tex

```diff
@@ -1 +1,21 @@
-\chapter{Inter Island Forecasting Organisation (IIFO)}
\ No newline at end of file
+\chapter{Inter Island Forecasting Organisation (IIFO)}
+
+The role of IIFO is to allow islands to make predictions about the likelihood and severity of risks in the event of natural disasters, which correlates to the long term collective risk dilemma, and the returns from foraging, which correlates to the short term collective risk dilemma.
+
+\section{Long Term Collective Risk Dilemma (ltCRD)}
+\label{sec:IIFO:ltCRD}
+
+Islands may build a model to try and predict the likelihood and severity of upcoming disasters. They are then able to share this with the other islands along with a confidence level for their prediction, allowing the island to specify how much they believe their prediction to be the truth so that in the case that the island is wrong, agents might not judge them as harshly if their confidence level was low. On the flipside, a higher confidence level indicates that other islands should potentially place more trust in this prediction with the risk that the island offering the information will be judged more harshly if they turn out to be wrong. 
+
+The motivation behind sharing this information is that with more people contributing to the common pool, the more the easily the effects of the disaster can be mitigated. Sharing predictions allows the islands to better manage their resources as they have a better idea of when they should be donated to the common pool or not.
```

Comment:
''people contributing''' :D  islands I gues...

suggested re-writing:
**The motivation behind sharing these predictions is that with more islands contributing to the common pool, the more greatly the effects of the disaster(s) can be mitigated.** **Therefore, sharing predictions allows islands to manage their resources in a more informed way, as they have a better idea of when and how much they should donate to the common pool in order to mitigate a future disaster.**